backend/src/packages/users/user.service.ts

```diff
@@ -10,9 +11,11 @@ import {
 
 class UserService implements Service {
   private userRepository: UserRepository;
+  private jwtService: JWTService;
 
-  public constructor(userRepository: UserRepository) {
+  public constructor(userRepository: UserRepository, jwtService: JWTService) {
```

Comment:
As we are using the approach of one param to constructor, let's proceed with it here

Reply:
I think I did it, that's a great Idea, thank you!